CONTRIBUTING.md

```diff
@@ -34,6 +34,27 @@ Some tips for hosting a great review club meeting:
   where to look in the codebase/GitHub/documentation, and let them do their
   own research from there.
 
+- When linking to code, use stable links that won't break later:
+
+  - for linking to code in the master branch, link to a recent commit hash in
+    the master branch of bitcoin/bitcoin, e.g.
+    [https://github.com/bitcoin/bitcoin/blob/23d3ae7a/src/addrman.cpp#L44](https://github.com/bitcoin/bitcoin/blob/23d3ae7a/src/addrman.cpp#L44).
+    You can use a short (8 character) commit id. REASON: linking directly to master
+    isn't stable because line numbers/files change over time as the HEAD commit of
+    master changes.
```

Comment:
perhaps omit "as the HEAD commit of master changes" or just "as the files change over time."